### PR TITLE
[TC-292] Update X8 Datasheet

### PR DESCRIPTION
**Pull request by aliphys**
## What This PR Changes
- Clearly state that accessories are not included 
- Removed unused components in current release
- State requirement of carrier board for the bootloader functionality

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md [hunk 1]
```diff
@@ -343,7 +305,7 @@ The Arduino® Portenta X8 has been designed for high performance embedded comput
 | P<sub>LP</sub>  | Power consumption in low power mode |      | 200  |      |  mW  |
 | P<sub>MAX</sub> | Maximum Power Consumption           |      | 4000 |      |  mW  |
 
-It is recommended to use a USB 3.0 port when connecting to the Portenta X8 which can deliver the required power. Dynamic scaling of the Portenta X8 can change the current consumption, leading to current surges during bootup. Average power consumption is provided in the above table for several reference scenarios.
+Use of a USB 3.0 compatible port will ensure that current requirements for the Portenta X8 are met. Dynamic scaling of the Portenta X8 compute units can change the current consumption, leading to current surges during bootup. Average power consumption is provided in the above table for several reference scenarios.
```
Comment (sebromero) [inline content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md:308]:
```suggestion
The use of a USB 3.0 compatible port will ensure that current requirements for the Portenta X8 are met. Dynamic scaling of the Portenta X8 compute units can change the current consumption, leading to current surges during bootup. Average power consumption is provided in the above table for several reference scenarios.
```

File: content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md [hunk 2]
```diff
@@ -435,7 +397,9 @@ Sample sketches for the Arduino® Portenta X8 can be found either in the “Exam
 Now that you have gone through the basics of what you can do with the board you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[5]**, the Arduino® Library Reference **[6]** and the online store **[7]** where you will be able to complement your board with sensors, actuators and more.
 
 ### Board Recovery
-All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by double-tapping the reset button right after power up.
+All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by configuring DIP switches. 
+
+***Note: A compatible carrier board with DIP switches (e.g. Portenta Max Carrier or Portenta Breakout) is required to enable bootloader mode. Bootloader cannot be reached with the Portenta X8 alone.***
```
Comment (sebromero) [inline content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md:402]:
```suggestion
**Note: A compatible carrier board with DIP switches (e.g. Portenta Max Carrier or Portenta Breakout) is required to enable bootloader mode. It cannot be enabled with the Portenta X8 alone.**
```

**Review by sebromero — CHANGES_REQUESTED** [review]
Just two small details, see suggestions. Otherwise looks good.

File: content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md (hunk 2 above)
Comment (aliphys) [reply content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md:402]: Should we use three asterisks or two for notes? @marqdevx

Comment (sebromero) [reply content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md:402]: @aliphys Three usually, but it won't be rendered as a note block inside the datasheets, only in the tutorials.

**Review by sebromero — APPROVED** [review]
LGTM

**Pull request merged**